app/src/main/java/com/example/dazn_movie_player/SplashScreen.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.dazn_movie_player
```

Comment:
Better not to have underscores for package name

Reply:
Use camel case notation